rai/node/lmdb.cpp

```diff
@@ -1884,15 +1885,24 @@ std::shared_ptr<rai::vote> rai::mdb_store::vote_generate (rai::transaction const
 	return result;
 }
 
-std::shared_ptr<rai::vote> rai::mdb_store::vote_max (rai::transaction const & transaction_a, std::shared_ptr<rai::vote> vote_a)
+std::shared_ptr<rai::vote> rai::mdb_store::vote_max (rai::transaction const & transaction_a, std::shared_ptr<rai::vote> vote_a, rai::vote_code & vote_result)
 {
 	std::lock_guard<std::mutex> lock (cache_mutex);
 	auto current (vote_current (transaction_a, vote_a->account));
 	auto result (vote_a);
+
+
 	if (current != nullptr && current->sequence > result->sequence)
```

Comment:
I guess this can be  
`current->sequence >= result->sequence`  
to prevent vote_code::vote result for current vote == new vote

Reply:
If current (DB/cache) is equal to the vote-in-question (result), we should yield "replay", right ?

Reply:
I think so. Otherwise old solution with == was enough